### PR TITLE
Follow-up fixes for static resources project tutorial

### PR DESCRIPTION
**Pull request by ebkalderon**
### Fixed

* Correct typo "eand" -> "and".

### Removed

* Remove noisy `result` line from filtering table.

Follow-up to #135 which addresses feedback given by @antigravityla on that PR.

### CODE REVIEW

File: examples/3-static-resources/README.md [hunk 1]
```diff
@@ -11,28 +11,6 @@ to only a few specific files and directories at build time. Specifically:
...
-│   └── main.rs
-├── tests
-│   └── my_tests.rs
-├── build.rs
-├── default.nix
-├── Cargo.lock
-├── Cargo.nix
-├── Cargo.toml
-├── README.md
-└── result -> /nix/store/qjz1m7pmm23r7fhgbigb2x5pnwqp925j-crate-foo-project-0.1.0
```
Comment (antigravityla) [inline examples/3-static-resources/README.md:29]: The nit I left was only for the `result` line.  The tree itself is helpful

Comment (ebkalderon) [reply examples/3-static-resources/README.md:29]: I see; my mistake! I can restore the tree without the `result` entry, in that case. Would that work, @antigravityla?

Comment (ebkalderon) [reply examples/3-static-resources/README.md:29]: Okay, amended the last commit!

**Comment by ebkalderon** [comment]
Anytime, @antigravityla! Thanks for the suggestions. Sorry I couldn't fit them into the other PR, but there were so many approvals already, I didn't want to invalidate them all for such minor changes. 😅

**Pull request merged**